lighthouse-core/test/runner-test.js

```diff
@@ -389,7 +390,7 @@ describe('Runner', () => {
     return Runner.run({}, {config}).then(results => {
       const audits = results.lhr.audits;
       assert.equal(audits['critical-request-chains'].displayValue, '5 chains found');
-      assert.equal(audits['critical-request-chains'].rawValue, false);
+      assert.equal(audits['critical-request-chains'].details.longestChain.transferSize, 2468);
```

Comment:
oh, I guess not quite string replace in this file because it made no sense to assert the score on these. They're all `null` since the audit is `INFORMATIVE`